pipeline/minimmit.md

```diff
@@ -212,13 +214,17 @@ _If you have already broadcast `notarize(c, v)` for a `c` that cannot be finaliz
 - Replicas enter `v + 1` as soon as they see some `proof(v)` (as fast as `L` messages). If the network is partitioned in two, replicas in each half of the partition may continue to enter successive views (on different `proof(v)`s) but will never finalize conflicting blocks. To bound the depth of forks in a partition, replicas can wait to enter some view `v + k` until they have seen `Q` messages in view `v`.
 - A Byzantine leader could equivocate, sending a distinct proposal to each replica and causing them to broadcast a `notarize(*, v)` for different blocks. After a replica observes `≥ L` `notarize(*, v)` messages for some `* != c`, it will then choose to broadcast a `nullify(v)` message. Eventually, `L` `nullify(v)` messages will be received and honest replicas will enter `v + 1` (within `Δ` of the first honest replica).
 - Since at most `f` nodes are Byzantine or faulty, once an honest leader is assigned, it is possible for at least `Q` correct replicas to finalize a block (including all of its ancestors).
+- A faulty leader can delay the network by at most `3Δ`. If replicas time out at `2Δ`, then a `nullification(v)` will be seen by replicas by time `3Δ`.
```

Comment:
I think we should measure time in relation to some `t0`? I find it tougher to reason about as just the duration.